Assets/Scripts/PlayerController.cs

```diff
@@ -35,12 +35,12 @@ public void UpdateCamera()
     // Update is called once per frame
     private void Update()
     {
-        if (Input.GetAxisRaw("Horizontal") != 0 || Input.GetAxisRaw("Vertical") != 0)
+        if (Input.GetAxis("Horizontal") != 0 || Input.GetAxis("Vertical") != 0)
         {
```

Comment:
Was this change intended?

Reply:
yea, it makes the turning more natural. I just put in a clause to make sure that the facing doesn't change if vector is zero, like what you mentioned previously.

Reply:
Okays! Just make sure that the character faces the direction he is moving towards. Perhaps we can add something like a "nose" for testing purposes.

Reply:
since we don't update only on the last frame, it should be facing the same direction as the last movement. If he is moving, then the facing is definitely on